browser/ui/views/frame/vertical_tab_strip_widget_delegate_view.cc

```diff
@@ -160,7 +160,7 @@ void VerticalTabStripWidgetDelegateView::UpdateClip() {
   SkPath path;
   const bool is_vertical_tab_left_most =
       !static_cast<BraveBrowserView*>(browser_view_)->IsSidebarVisible() ||
-      !browser_view_->browser()->profile()->GetPrefs()->GetBoolean(
+      browser_view_->browser()->profile()->GetPrefs()->GetBoolean(
           prefs::kSidePanelHorizontalAlignment);
```

Comment:
Fix another bug together - When kSidebarPanelHorizontalAlignment is true, sidebar is on the right.